platform-integrations/claw-code/plugins/evolve-lite/skills/save-trajectory/SKILL.md

```
@@ -0,0 +1,156 @@
+Write the trajectory JSON to a temporary file using the **Write** tool, then pass the file path to the helper script:
+
+1. Write the JSON to `.evolve/tmp/trajectory_input.json` using the Write tool (create the directory if needed)
+2. Run the helper script with the file path as an argument:
+
+```bash
+tmp=.evolve/tmp/trajectory_input.json; mkdir -p .evolve/tmp; trap 'rm -f "$tmp"' EXIT; script=".claw/skills/evolve-lite:save-trajectory/scripts/save_trajectory.py"; [ -f "$script" ] || script="$HOME/.claw/skills/evolve-lite:save-trajectory/scripts/save_trajectory.py"; python3 "$script" "$tmp"
+```
+
+**Important**: Do NOT use inline Python scripts, heredocs, or stdin piping to pass the trajectory JSON. Always use the Write tool to create a temp file first. This avoids escaping issues with backslashes, quotes, and newlines in conversation content.
+
+### Step 5: Save via Helper Script
```

markdownlint-cli2 (0.22.0)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)